.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,28 @@
+name: Continuous Integration
+
+on:
+  push:
```

Comment:
I think we want to support `pull_request` event too. Or `pull_request_target` if it involves aws credentials